cirq-core/cirq/ops/raw_types.py

```diff
@@ -864,6 +868,13 @@ def without_classical_controls(self) -> 'cirq.Operation':
         new_sub_operation = self.sub_operation.without_classical_controls()
         return self if new_sub_operation is self.sub_operation else new_sub_operation
 
+    def with_classical_controls(
```

Comment:
I think that's fine, and also consistent with other operations like controlled ops. The general guidance would also be to use `op.with_classical_controls` instead of using the constructor.

Reply:
sg, I guess the `without` docstring doesn't need updated then, unless there's a need to be explicit that that's the only case in which a TaggedOp could be returned. (`with` still needs updated though).